docs/guides/configure-error-mitigation.mdx

```diff
@@ -58,7 +58,9 @@ applied at each resilience level.
 
 ## Configure Estimator with resilience levels
 
-You can use resilience levels to specify error mitigation techniques, or you can set custom techniques individually as described in [Custom error settings.](#advanced-error) You cannot specify resilience levels in Sampler. However, you can set custom techniques individually.
+You can use resilience levels to specify error mitigation techniques, or you can set custom techniques individually as described in [Custom error settings.](#advanced-error) 
+
+Sampler does not support error mitigation, but you can use the [mthree](https://qiskit.github.io/qiskit-addon-mthree/) (matrix-free measurement mitigation) package to do error mitigation locally.
```

Comment:
linking to the ecosystem docs here - could link to the github repo instead or in addition - lmk if there is a preference for one of those options.

Reply:
I find it a bit odd to see this paragraph under `Configure Estimator with resilience levels`, since this is not related to resilience levels. However, I cannot decide if it's better under `Custom error settings, since the code example for Sampler there shows error _suppression_, not mitigation.

Reply:
I can't figure out how to add a comment to a part of the file that wasn't edited, but maybe this would work: 
1 - At the top where we say "Primitives support several error mitigation techniques...", change that to "Estimator supports..." (since sampler doesn't support any)
2 - Move this m3 content after that updated paragraph as a note.

Reply:
I was going to suggest it being a note too. @jyu00 do you have a preference for one of Becky's suggestions?

Reply:
I think Becky was suggesting we do both? Change the top to say just Estimator, and follow that with a note saying you can use M3 for Sampler. I do like this approach.

Reply:
ah yes, I don't know why I assumed it was one or the other 🤣 both it is!